src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
This interface itself requires the header comment, so that others that try to read it will understand the purpose of this interface.